origins_workflow/origins_workflow.module

```diff
@@ -413,3 +414,48 @@ function _create_draft_link_for_revision(&$variables, $idx, $row) {
   }
   return $this_version;
 }
+
+/**
+ * Implements hook_moderation_sidebar_alter().
+ */
+function origins_workflow_moderation_sidebar_alter(array &$build, EntityInterface $entity) {
+  // Get current moderation state.
+  $current_state = $entity->moderation_state->value;
+  // Special case if this node is published but
+  // we are not looking at the latest revision.
+  if (($current_state == 'published') && !$entity->isLatestRevision()) {
```

Comment:
`($current_state == 'published')` can have the parentheses dropped, because in the case of AND logic if the first condition fails (left to right) then it doesn't bother evaluating the second as the first failed condition forces a FALSE value for the expression as a whole.